torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -4137,3 +4142,53 @@ def test_device_maps_remote(self):
         )
 
         self.assertEqual(rref.to_here(), torch.ones(2).to(1))
+
+    @requires_nccl()
+    @skip_if_lt_x_gpu(4)
+    def test_with_ddp(self):
```

Comment:
I'm not familiar with these tests: what is the reason to add this to the `rpc_test` file rather than to the `ddp_under_dist_autograd_test` file?